smoke-tests/SmokeTestGenerator.js

```diff
@@ -45,43 +45,67 @@ function buildPath(pathSegs, delim) {
 function generateScriptBody(isBash, delim) {
 	const projectsPath = ['test', 'code-fixtures', 'projects'];
 	const customRulePath = ['test', 'test-jars', 'apex', 'testjar1.jar'];
+	const customConfigPath = ['test', 'test-xml', 'apex', 'smoke-config.xml'];
 	const resultsPath = ['smoke-test-results'];
 
 	const exeName = isBash ? '$EXE_NAME' : '%EXE_NAME%';
 
 	// Declare an array with all of the commands we intend to execute.
 	const commands = [
+		// Log a header.
```

Comment:
Added commenting between lines in order to more clearly delineate different stages.